docs/docs-content/getting-started/aws/scale-secure-cluster.md

```diff
@@ -0,0 +1,505 @@
+security of your clusters. Expand the following section to learn how to create a new cluster profile version with a
+Kubernetes upgrade.
+
+<details>
+
+<summary>Upgrade Kubernetes using Cluster Profile Versions</summary>
+
+Navigate to the left **Main Menu** and click on **Profiles**. Select the cluster profile that you used to deploy your
+cluster, named **aws-profile**. The cluster profile details page appears.
+
+Click on the version drop-down and select **Create new version**. The version creation dialog appears.
+
+Fill in **1.1.0** in the **Version** input field. Then, click on **Confirm**. The new cluster profile version is created
+with the same layers as version **1.0.0**.
+
+Select the **kubernetes 1.27.x** layer of the profile. The pack manifest editor appears.
+
+Click on the **Pack Version** dropdown. All of the available versions of the **Palette eXtended Kubernetes** pack
+appear. The cluster profile is configured to use the latest patch version of **Kubernetes 1.27**.
+
+![Cluster profile with all Kubernetes versions](/getting-started/aws/getting-started_scale-secure-cluster_kubernetes-versions.webp)
+
+The official guidelines for Kubernetes upgrades recommend upgrading one minor version at a time. For example, if you are
+using Kubernetes version 1.26, you should upgrade to 1.27, before upgrading to version 1.28. You can learn more about
+the official Kubernetes upgrade guidelines in the
+[Version Skew Policy](https://kubernetes.io/releases/version-skew-policy/) page.
+
+Select **1.28.x** from the version dropdown. This selection follows the Kubernetes upgrade guidelines as the cluster
+profile is using **1.27.x**.
+
+The manifest editor highlights the changes made by this upgrade. Once you have verified that the upgrade changes
+versions as expected, click on **Confirm changes**.
+
+Click on **Confirm Updates**. Then, click on **Save Changes** to persist your updates.
+
+Navigate to the left **Main Menu** and select **Clusters**. Select your cluster to view its **Overview** tab.
+
+Select the **Profile** tab. Your cluster is currently using the **1.0.0** version of your cluster profile.
+
+Change the cluster profile version by selecting **1.1.0** from the version drop-down. Click on **Review & Save**. The
+**Changes Summary** dialog appears.
+
+Click on **Review changes in Editor**. The **Review Update Changes** dialog displays the same Kubernetes version
+upgrades as the cluster profile editor previously did. Click on **Update**.
+
+</details>
+
+Upgrading the Kubernetes version of your cluster modifies an infrastructure layer. Therefore, Kubernetes needs to
+replace its nodes. This is known as a repave. Check out the
+[Node Pools](../../clusters/cluster-management/node-pool.md#repave-behavior-and-configuration) page to learn more about
+repave behavior and configuration.
+
+Click on the **Nodes** tab. You can follow along with the node upgrades on this screen. Palette replaces the nodes
```

Comment:
Do we have downtime when Palette replaces the nodes?

Reply:
Added a note that there are some downtimes while the workloads start up on the new node.